README.md

```diff
@@ -182,5 +189,8 @@ Godspeed!
   [Ansible Vault]:http://docs.ansible.com/ansible/playbooks_vault.html
   [inventory]:https://docs.ansible.com/ansible/intro_inventory.html
   [slacktee]:https://github.com/course-hero/slacktee
+  [ansible]:https://github.com/ansible/ansible
   [deployment-new-production]:docs/deployment-new-production.md
   [IT Crowd]:mailto:itcrowd@svsticky.nl
+  [deployment-guide]:https://github.com/svsticky/sadserver/blob/master/README.md#setting-up-the-staging-and-production-environment
```

Comment:
If you leave out everything before the anchor, the link will also work on other branches, like this one ;)

Reply:
that is `[deployment-guide](setting-up-the-staging-and-production-environment)` by the way :P